ledger/eval_simple_test.go

```diff
@@ -224,92 +224,84 @@ func TestMiningFees(t *testing.T) {
 		dl := NewDoubleLedger(t, genBalances, cv, cfg)
 		defer dl.Close()
 
-		tooBig := basics.Address{0x01, 0x011}
-		tooSmall := basics.Address{0x01, 0x022}
-		smallest := basics.Address{0x01, 0x033}
-		biggest := basics.Address{0x01, 0x044}
-
+		proposer := basics.Address{0x01, 0x011}
 		const eFee = 3_000_000
-		dl.txns(
-			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: tooBig, Amount: eFee + 100_000_000*1_000_000 + 1},
-			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: tooSmall, Amount: eFee + 100_000*1_000_000 - 1},
-			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: smallest, Amount: eFee + 100_000*1_000_000},
+		dl.txn(
 			&txntest.Txn{Type: "pay", Sender: addrs[1],
-				Receiver: biggest, Amount: eFee + 100_000_000*1_000_000},
+				Receiver: proposer, Amount: eFee + 50_000_000*1_000_000 + 1},
 		)
 
-		for _, proposer := range []basics.Address{tooBig, tooSmall, smallest, biggest} {
-			t.Log(proposer)
+		prp := lookup(dl.t, dl.generator, proposer)
+		require.False(t, prp.IncentiveEligible)
 
-			prp := lookup(dl.t, dl.generator, proposer)
-			require.False(t, prp.IncentiveEligible)
+		dl.txn(&txntest.Txn{
+			Type:         "keyreg",
+			Sender:       proposer,
+			Fee:          eFee,
+			VotePK:       crypto.OneTimeSignatureVerifier{0x01},
+			SelectionPK:  crypto.VRFVerifier{0x02},
+			StateProofPK: merklesignature.Commitment{0x03},
+			VoteFirst:    1, VoteLast: 1000,
+		})
 
-			dl.txn(&txntest.Txn{
-				Type:         "keyreg",
-				Sender:       proposer,
-				Fee:          eFee,
-				VotePK:       crypto.OneTimeSignatureVerifier{0x01},
-				SelectionPK:  crypto.VRFVerifier{0x02},
-				StateProofPK: merklesignature.Commitment{0x03},
-				VoteFirst:    1, VoteLast: 1000,
-			})
+		prp = lookup(dl.t, dl.generator, proposer)
+		require.Equal(t, ver >= miningBegins, prp.IncentiveEligible)
 
-			prp = lookup(dl.t, dl.generator, proposer)
-			require.Equal(t, ver >= miningBegins, prp.IncentiveEligible)
+		dl.fullBlock() // start with an empty block, so no mining fees are paid at start of next one
 
-			dl.fullBlock() // start with an empty block, so no mining fees are paid at start of next one
+		presink := micros(dl.t, dl.generator, genBalances.FeeSink)
+		preprop := micros(dl.t, dl.generator, proposer)
+		t.Log(" presink", presink)
+		t.Log(" preprop", preprop)
+		dl.beginBlock()
+		pay := txntest.Txn{
+			Type:     "pay",
+			Sender:   addrs[1],
+			Receiver: addrs[2],
+			Amount:   100000,
+		}
+		dl.txns(&pay, pay.Args("again"))
+		vb := dl.endBlock(proposer)
+
+		const bonus1 = 5_000_000 // the first bonus value, set in
```

Comment:
set in the consensus bonus plan?

Reply:
Correct, this is one of those things were I like to use the numeric constant so that the unit tests fail if suddenly changed in the consensus protocol.